scripts/wait_for_server.sh

```diff
@@ -1,20 +1,23 @@
 #!/usr/bin/env bash
 
-counter=0
 status=""
+counter=0
+checkcounter=0
```

Comment:
Just a small nitpick, but why do we need two counters?

Reply:
It's this way on Files :shrug:  I can remove it from both if needed

Reply:
All changes except wait time increase are now undone

Reply:
Hm. Strange, I have no idea why this is needed.
At some point we shall move this also to common…